frame/support/src/traits/hooks.rs

```diff
@@ -336,26 +354,38 @@ pub trait Hooks<BlockNumber> {
 		Ok(())
 	}
 
-	/// Implementing this function on a module allows you to perform long-running tasks
-	/// that make (by default) validators generate transactions that feed results
-	/// of those long-running computations back on chain.
+	/// Implementing this function on a pallet allows you to perform long-running tasks that are
+	/// dispatched as separate threads, and entirely independent of the main wasm runtime.
```

Comment:
```suggestion
	/// executed as separate threads, and entirely independent of the main state transition function (even though its functionality is also defined by the runtime wasm blob).
```

this comes back to what we define as "runtime": wasm blob or STF?